src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -11,7 +11,12 @@
       "title": "Diagnostic Level",
       "description": "Type of diagnostic to display, most rules default to warning",
       "type": "string",
-      "enum": ["off", "info", "warning", "error"]
+      "enum": [
+        "off",
+        "info",
+        "warning",
+        "error"
+      ]
```

Comment:
Can we restore the formatting? The file was formatted with prettier to make sure it's consistent for everyone. I'll need to add new eslint plugin to enforce that later.